conans/test/unittests/tools/google/test_bazeldeps.py

```diff
@@ -34,8 +40,11 @@ def test_bazeldeps_dependency_buildfiles():
         for dependency in bazeldeps._conanfile.dependencies.host.values():
             dependency_content = bazeldeps._get_dependency_buildfile_content(dependency)
             assert 'cc_library(\n    name = "OriginalDepName",' in dependency_content
-            assert 'defines = ["DUMMY_DEFINE=\'string/value\'"],' in dependency_content
-            assert 'linkopts = ["-lsystem_lib1"],' in dependency_content
+            assert """defines = ["DUMMY_DEFINE=\\\\\\"string/value\\\\\\""]""" in dependency_content
```

Comment:
So many backslashes necessary? Quite ugly...

Reply:
Just the half of them, the rest is to escape the assert. 🤣 
The rest are absolutely necessary.